finance.reckoner.app.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: finance.reckoner.app
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+command: reckoner
+separate-locales: false
+finish-args:
+  [
+    --socket=x11,
+    --socket=wayland,
+    --share=network,
+    --filesystem=home,
```

Comment:
why do you need home? Does it support filechooser portals?

Reply:
Yes, this supports exporting the database using a filepicker. The home directory access seemed most appropriate for this.

Reply:
If the filepicker uses portals you don't need any filesystem permission

Reply:
I tested and yes I need home access. I am not using the portals and trying to get that working with Flutter is beyond the scope of what I want to add to the app.